### ISSUE TITLE
WAR file not generated in 0.5.0 version

### ISSUE THREAD

**Issue by ngbabu** [kermitt2/grobid#272]
I have built the 0.5.0 version using Gradle. which got replaced from Maven. In this new version War file not got generated to deploy into Tomcat. Please add the option to build WAR file in Gradle.

**Comment by kermitt2** [comment]
Indeed, generating a war is important for a java web application, we will need to add graddle war plugin.

**Comment by ngbabu** [comment]
WAR file is getting generated. But we can not deploy this file on Tomcat as the following folders and files are missing when compared to previous version WAR file.

1. web.xml
2. resource folder
3. grobid folder with grobid.js
4. index.html

**Comment by lfoppiano** [comment]
@ngbabu my bad, I was doing some experiments in order to produce a new docker file and I forgot the plugin in the build.gradle. I'm going to comment it (you can still unconmment it and try to make it work)

Unfortunately there is more to be done than just that ..  we need to figure out also which dependencies to add for the servlets and make sure we don't break the current package.

**Comment by lfoppiano** [comment]
This is obsolete, should be closed. Since there is no war anymore in the picture here.
